driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -232,11 +234,16 @@ fn write_input(input_file: &str, input: &str) -> std::io::Result<()> {
     Ok(())
 }
 
-fn run_solver(input_file: &str) -> Result<(), PriceFindingError> {
+fn run_solver(
+    input_file: &str,
+    optimization_model: OptimizationModel,
+) -> Result<(), PriceFindingError> {
+    let optimization_model_str = optimization_model.to_args()?;
     let output = Command::new("python")
         .args(&["-m", "batchauctions.scripts.e2e._run"])
         .arg(RESULT_FOLDER)
         .args(&["--jsonFile", input_file])
+        .args(&["--optModel", optimization_model_str])
```

Comment:
Going forward `to_args` could return the full arguments it needs in case the configuration cannot just be done with a single `--optModel` parameter